includes/admin/reporting/export/class-batch-export-payments.php

```diff
@@ -195,8 +195,7 @@ public function get_data() {
 				'seq_id'       => $order->get_number(),
 				'email'        => $order->email,
 				'customer_id'  => $order->customer_id,
-				'first'        => isset( $address->first_name ) ? $address->first_name : '',
-				'last'         => isset( $address->last_name ) ? $address->last_name : '',
+				'name'         => $customer->name,
```

Comment:
```suggestion
				'name'         => ! empty( $customer->name ) ? $customer->name : '',
```

In case `$customer` didn't actually have a result.